### PR TITLE
wip packaging

### PR DESCRIPTION
**Pull request by jelbourn**

### CODE REVIEW

File: src/lib/snack-bar/index.ts [hunk 1]
```diff
@@ -7,7 +8,13 @@ import {SimpleSnackBar} from './simple-snack-bar';
 
 
 @NgModule({
-  imports: [OverlayModule, PortalModule, CommonModule, CompatibilityModule],
+  imports: [
+    BrowserAnimationsModule,
```
Comment (IgorMinar) [inline src/lib/snack-bar/index.ts]: same as above..

File: package.json [hunk 2]
```diff
@@ -25,23 +25,23 @@
...
   },
   "dependencies": {
-    "@angular/animations": "^4.0.0-rc.3",
-    "@angular/common": "^4.0.0-rc.3",
-    "@angular/compiler": "^4.0.0-rc.3",
-    "@angular/core": "^4.0.0-rc.3",
-    "@angular/forms": "^4.0.0-rc.3",
-    "@angular/http": "^4.0.0-rc.3",
-    "@angular/platform-browser": "^4.0.0-rc.3",
+    "@angular/animations": "^4.0.0-rc.5",
```
Comment (IgorMinar) [inline package.json]: nit: squash this commit with the other one?

File: package.json [hunk 3]
```diff
@@ -99,16 +99,17 @@
...
     "selenium-webdriver": "^3.1.0",
     "stylelint": "^7.8.0",
     "travis-after-modes": "0.0.7",
-    "ts-node": "^2.1.0",
+    "ts-node": "^3.0.0",
     "tslint": "^4.4.2",
     "tslint-no-unused-var": "0.0.6",
-    "typescript": "~2.1.6",
-    "uglify-js": "^2.8.7",
+    "typescript": "~2.2.1",
```
Comment (IgorMinar) [inline package.json]: fyi: just be aware that google3 currently runs on 2.1.x, 2.2.x upgrade is in works.

File: src/demo-app/tsconfig-aot.json [hunk 4]
```diff
@@ -25,7 +12,6 @@
     "./main-aot.ts"
   ],
   "angularCompilerOptions": {
-    "skipTemplateCodegen": false,
-    "debug": true
+    "skipTemplateCodegen": false
```
Comment (IgorMinar) [inline src/demo-app/tsconfig-aot.json]: isn't this set to false by default ?

**Comment by jelbourn** [comment]
TODO:
- [ ] Fix AOT errors on CI
- [ ] Add package.json to `release/`
- [ ] Fix `release/index.d.ts` and `release/public_api.d.ts` pointing to the wrong paths (likely just removing typings for now)
- [ ] Inline resources in metadata.json
- [ ] Source maps (bonus)

**Comment by jelbourn** [comment]
cc @DevVersion

**Pull request closed without merging**